### PR TITLE
Feature/8395 qrscanner/9388 new UI scanner

### PR DESCRIPTION
**Pull request by pascal-brause**
## Description
Following PR for #3529. The original PR was closed due to branching switch so I reopened this PR for @naveeddotio.

## Screenshots
WhatsApp Image 2021-09-16 at 16 16 40
WhatsApp Image 2021-09-16 at 16 16 40 (1)

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift [hunk 1]
```diff
@@ -2,20 +2,322 @@
...
+		navigationController?.navigationBar.isTranslucent = true
+		navigationController?.view.backgroundColor = .enaColor(for: .background)
+
+		let cancelItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(didPressDismiss))
+		cancelItem.accessibilityIdentifier = AccessibilityIdentifiers.General.cancelButton
+		navigationItem.leftBarButtonItem = cancelItem
+	}
+
+	@objc
+	private func didPressDismiss() {
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift:173]:
```suggestion
	private func didTapDismiss() {
```

**Review by pascal-brause — COMMENTED** [review]
Please remove all TODOs and add the implementation. Therefore you can merge #3535 into your branch so can access it's content.

Also check again for transparency of the white background. In Figma its defined transparent.

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Review by pascal-brause — COMMENTED** [review]
Because i recreated the PR for Naveed, I cannot approve it. So Naveed approve it for me :)

**Review by naveeddotio — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Pull request merged**